### ISSUE TITLE
possible for loop bug

### ISSUE THREAD

**Issue by bdwyer2** [ravidziv/IDNNs#4]
In [idnns/information/information_process.py](https://github.com/ravidziv/IDNNs/blob/master/idnns/information/information_process.py) on [line 172](
https://github.com/ravidziv/IDNNs/blob/master/idnns/information/information_process.py#L172) there is a `for` loop using the variable `i` however we are already in another `for` loop also using `i` starting on [line 166](
https://github.com/ravidziv/IDNNs/blob/master/idnns/information/information_process.py#L172).

Is this a bug or was is it intended?

Lines 166 - 178:
```python
        for i in range(len(ws_iter_index)):
            data = ws_iter_index[i]
            new_data = np.zeros((num_of_samples * data.shape[0], data.shape[1]))
            labels = np.zeros((num_of_samples * label.shape[0], label.shape[1]))
            x = np.zeros((num_of_samples * data.shape[0], 2))
            sigma = 0.5
            for i in range(data.shape[0]):
                print i
                cov_matrix = np.eye(data[i, :].shape[0]) * sigma
                t_i = np.random.multivariate_normal(data[i, :], cov_matrix, num_of_samples)
                new_data[num_of_samples * i:(num_of_samples * (i + 1)) , :] = t_i
                labels[num_of_samples * i:(num_of_samples * (i + 1)) , :] = label[i, :]
                x[num_of_samples * i:(num_of_samples * (i + 1)) , 0] = i
```

**Comment by ravidziv** [comment]
It's a bug. I fixed it. thanks!

**Comment by bdwyer2** [comment]
Thanks for the quick fix!

Issue resolved via c7b75c216f145745bb223dbb4575a341f73002a3
